### DISCUSSION TITLE
when i setup budibase repo on windows and run on this port localhost:3000 : getting not found

### DISCUSSION THREAD

**Discussion by bhavin-codup** [Budibase/budibase#13605] [Contributing]
After Enter the Email, password and Confirm Password  and hit button on signup  
i am getting an error like not found 
![image](https://github.com/Budibase/budibase/assets/153892424/d367ef17-9e59-45f3-9a5b-bb2d9acaaf42)

**Comment by mike12345567** [comment]
Hi @bhavin-codup - that is the port of the Vite server, however the development server should be proxied through port `10000` by default, you will have an nginx proxy server running in Docker, alongside the Redis, MinIO and CouchDB images which are needed.

**Reply by bhavin-codup** [reply]
when i working on port 10000 and changes in source UI and refresh the browser it's not reflecting.

can you please help me for reflect the UI when i change in file's.
